web_ir_actions_act_multi/models/ir_actions.py

```diff
@@ -7,7 +7,7 @@ class IrActionsActMulti(models.Model):
     _name = "ir.actions.act_multi"
     _description = "Action Mulit"
     _inherit = "ir.actions.actions"
-    _table = "ir_actions"
+    _table = "ir_act_multi"
```

Comment:
I think you should better do this with an uninstall hook to avoid having another relation. And maybe this could have migration issues when updating the module.